### PR TITLE
do not cluster periodic pinging

### PR DESCRIPTION
**Pull request by mafintosh**

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -111,9 +113,15 @@ DHT.prototype._setBucketCheckInterval = function () {
           // retry with initial bootstrap nodes
           self._bootstrap(true)
         }
+        
+        queueNext()
```
Comment (robertkowalski) [inline client.js]: if it is in the if, the interval will stall as soon as the bucket is not outdated (15min)

`if (diff >= self._bucketOutdatedTimeSpan) {`

Comment (mafintosh) [reply client.js]: ah right, nice catch

**Comment by robertkowalski** [comment]
works like a charm +1 <3

**Comment by robertkowalski** [comment]
ah wait!

**Pull request merged**